Libs/Common/Shapeworks.h

```diff
@@ -261,9 +261,12 @@ bool epsEqual(const P &a, const P &b, const typename P::ValueType &eps)
 
 // https://stackoverflow.com/a/17382806/207044
 template<typename P>
-bool equalNSigDigits(P a, P b, int n = 4)
+bool equalNSigDigits(P a, P b, int n = 4, double eps = 0.0)
```

Comment:
There is already an epsEqual function in shapeworks.h. We do not want to change this one.

Reply:
Removed it